src/jira/client/index.ts

```diff
@@ -281,20 +281,27 @@ const batchedBulkUpdate = async (
 	options?: { preventTransitions: boolean }
 ) => {
 	// Initialize with an empty chunk of commits so we still process the request if there are no commits in the payload
-	const commitChunks: JiraCommit[][] = _.chunk(dedupCommits(data.commits), 400);
+	let commitChunks: JiraCommit[][] = _.chunk(dedupCommits(data.commits), 400);
```

Comment:
Let's add a feature flag, so there's no functional change when the code goes to production

Reply:
:+1: